execution/gethexec/caff_node.go

```diff
@@ -81,12 +90,32 @@ func NewCaffNode(configFetcher SequencerConfigFetcher, execEngine *ExecutionEngi
 		return nil
 	}
 
+	db, err := stack.OpenDatabase("caffdata", 0, 0, "caffdata/", false)
+	if err != nil {
+		log.Crit("could not open the caff database")
+		return nil
+	}
+
+	currentPosition, nextHotshotBlock, err := ReadCurrentPositionAndNextHotshotBlockFromDb(db)
```

Comment:
note: we dont need to store the currentPosition only the hotshot block number because currentPos is stored in the block database